wdn/templates_4.1/scripts/js-css/script-font.less

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license
+ * MyFonts Webfont Build ID 3193178, 2016-03-28T11:06:51-0400
+ *
+ * The fonts listed in this notice are subject to the End User License
+ * Agreement(s) entered into by the website owner. All other parties are
+ * explicitly restricted from using the Licensed Webfonts(s).
+ *
+ * You may obtain a valid license at the URLs below.
+ *
+ * Webfont: Sanelma by Mika Melvas
+ * URL: http://www.myfonts.com/fonts/mika-melvas/sanelma/regular/
+ * Copyright: Copyright (c) 2014 by Mika Melvas. All rights reserved.
+ * Licensed pageviews: 10,000,000
+ *
+ *
+ * License: http://www.myfonts.com/viewlicense?type=web&buildid=3193178
+ *
+ * © 2016 MyFonts Inc
+*/
+
+/* @import must be at top of file, otherwise CSS will not work */
+@import url("//hello.myfonts.net/count/30b95a");
```

Comment:
Another coding standards practice I want to see adopted is _always_ using `https` when available, rather than the `//` scheme relative URLs. This has already been implemented in most other framework files.